euphoria-operator-testkit/src/main/java/cz/seznam/euphoria/operator/test/ReduceByKeyTest.java

```diff
@@ -747,11 +747,11 @@ public void validate(Partitions<Integer> partitions) {
 
         ArrayList<Long> triggerTimes = new ArrayList<>(TETETS_SEEN_TIMES_TRIGGER);
         triggerTimes.sort(Comparator.naturalOrder());
-        assertEquals(asList(15_000L, 19_999L, 25_000L, 29_999L), triggerTimes);
+        assertEquals(asList(19_999L, 19_999L, 29_999L, 29_999L), triggerTimes);
 
         ArrayList<Long> assignerTimes = new ArrayList<>(TETETS_SEEN_TIMES_ASSIGNER);
         assignerTimes.sort(Comparator.naturalOrder());
-        assertEquals(asList(15_000L, 19_999L, 25_000L, 29_999L), assignerTimes);
+        assertEquals(asList(19_999L, 19_999L, 29_999L, 29_999L), assignerTimes);
```

Comment:
not sure we need this test anymore. i think `testElementTimestamp` should suffice.